src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -251,7 +237,6 @@ void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aC
     {
         chip::TLV::Debug::Dump(aReader, TLVPrettyPrinter);
     }
-    gWaitingForCommandResp = false;
```

Comment:
Don't you need to call `notify_one` in the function?

Reply:
Good catch, thanks!